operator/v1/0000_70_cluster-network-operator_01_crd.yaml

```diff
@@ -307,6 +307,12 @@ spec:
                                 type: integer
                                 format: int32
                                 minimum: 0
+                        hybridOverlayVXLANPort:
```

Comment:
@openshift/openshift-team-sdn is this expected?

Reply:
That was added recently (#678). I guess some files didn't get regenerated?